cmd/panicbot/main.go

```diff
@@ -19,8 +20,81 @@ import (
 // 	watchFile(filePath string) error
 // }
 type Config struct {
-	DiscordBotToken string
-	GuildID         string
+	DiscordBotToken  string
+	GuildID          string
+	PrimaryChannelID string
+	AlertingMethods  *AlertingMethods
+	Voting           *Voting
+}
+type AlertingMethods struct {
+	Twilio struct {
+		AccountSID        string
+		AuthToken         string
+		TwilioPhoneNumber string
+	}
+	Email struct {
+		Auth struct {
+			Indentity string
+			Username  string
+			Password  string
+			Host      string
+		}
+		From           string
+		DefaultMessage string
+	}
+}
+type Voting struct {
+	MaxBannablePerVote int
+	RequiredVotes      struct {
+		PanicAlert int
+		PanicBan   int
+	}
+	ContactOnVote struct {
+		Discord struct {
+			Users []string
+			Roles []string
+		}
+		Twilio struct {
+			PhoneNumbers []string
+		}
+		Email struct {
+			Addresses []string
+		}
+	}
+	AllowedToVote struct {
+		PanicAlert struct {
+			Users []string
+			Roles []string
+		}
+		PanicBan struct {
+			Users []string
+			Roles []string
+		}
+	}
+	VoteTimers struct {
+		PanicBanVoteTimer   int
```

Comment:
duration strings

Reply:
I think what @deefdragon meant here is to convert the data type to a string instead of a time.Duration.
Because when you have a string you can pass it to the time.ParseDuration method to get a Duration object that can be used later on in Go application code.

https://pkg.go.dev/time#ParseDuration @Xanzibaer